### ISSUE TITLE
Print IGN base layer

### ISSUE THREAD

**Issue by Mavialle** [3liz/lizmap-web-client#1231]
Hello, 

To print OSM baselayer, I know you can use an xml file (see [here](https://docs.lizmap.com/current/fr/publish/advanced_lizmap_config.html#allow-printing-of-external-baselayers)). Is there a similar way to do that with IGN baselayer ? Or the only way is to use IGN raster data (usually heavy files, and I'm afraid it'll take a lot of space on the server).

Thank you in advance for your assistance.

Mavialle

**Comment by mdouchin** [comment]
Thanks for reporting the solution. Do you think you could help us improve the related documentation here ?
https://github.com/3liz/lizmap-documentation/

**Comment by Mavialle** [comment]
Yes, I will do it. Is it possible to contribute in french or is it english only ?
